book_app/reviews/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from reviews import views
+
+
+router = DefaultRouter()
+router.register('reviews', views.BookReviewViewSet)
+
+app_name = 'review'
+base_name = 'review'
```

Comment:
base_name should be in router.register. Now that all of this is working you can remove it.